src/components/StoreBadge/StoreBadge.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import block from 'bem-cn-lite';
+
+import {Link, Platform, Icon, LinkProps} from '@gravity-ui/uikit';
+
+import {GooglePlayEn} from '../../icons/stores/GooglePlayEn';
+import {GooglePlayRu} from '../../icons/stores/GooglePlayRu';
+import {AppStoreEn} from '../../icons/stores/AppStoreEn';
+import {AppStoreRu} from '../../icons/stores/AppStoreRu';
+
+import {Lang} from '../utils/configure';
+import {i18n} from '../../i18n';
+
+const b = block('store-badge');
+
+export type BadgePlatform = Platform.ANDROID | Platform.IOS;
+
+export interface StoreBadgeProps extends Omit<LinkProps, 'view'> {
+    platform: BadgePlatform;
+}
+
+const badgeData: Record<BadgePlatform, Record<Lang, unknown>> = {
+    [Platform.IOS]: {
+        ru: AppStoreRu,
+        en: AppStoreEn,
+    },
+    [Platform.ANDROID]: {
+        ru: GooglePlayRu,
+        en: GooglePlayEn,
+    },
+};
+
+/**
+ * Component for rendering store badges
+ *
+ * @param platform - platform name
```

Comment:
There are a lot of props in object. I think u can skip @param and @return parts

Reply:
I drop jsdoc, because eslint requires **@return**